addons/l10n_lt/data/account_fiscal_position_template_data.xml

```diff
@@ -0,0 +1,55 @@
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_id" ref="base.lt"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_fiscal_position_template_eu" model="account.fiscal.position.template">
+        <field name="name">EU</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_group_id" ref="base.europe"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="20"/>
+    </record>
+    <record id="account_fiscal_position_template_b2c_eu" model="account.fiscal.position.template">
+        <field name="name">B2C EU</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="False"/>
+        <field name="country_group_id" ref="base.europe"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="25"/>
+    </record>
+    <record id="account_fiscal_position_template_out_eu" model="account.fiscal.position.template">
+        <field name="name">Not EU</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="30"/>
+    </record>
+    <!-- Account Fiscal Position Tax Template -->
+    <record id="account_fiscal_position_tax_template_eu_1" model="account.fiscal.position.tax.template">
+        <field name="position_id" ref="account_fiscal_position_template_eu"/>
+        <field name="tax_src_id" ref="account_tax_template_sales_21"/>
+        <field name="tax_dest_id" ref="account_tax_template_sales_0_vat13"/>
+    </record>
+    <record id="account_fiscal_position_tax_template_eu_2" model="account.fiscal.position.tax.template">
+        <field name="position_id" ref="account_fiscal_position_template_eu"/>
+        <field name="tax_src_id" ref="account_tax_template_purchase_21"/>
+        <field name="tax_dest_id" ref="account_tax_template_purchase_assumed_21"/>
+    </record>
+    <record id="account_fiscal_position_tax_template_out_eu_1" model="account.fiscal.position.tax.template">
+        <field name="position_id" ref="account_fiscal_position_template_out_eu"/>
+        <field name="tax_src_id" ref="account_tax_template_sales_21"/>
+        <field name="tax_dest_id" ref="account_tax_template_sales_0_vat12"/>
+    </record>
+    <record id="account_fiscal_position_tax_template_out_eu_2" model="account.fiscal.position.tax.template">
+        <field name="position_id" ref="account_fiscal_position_template_out_eu"/>
+        <field name="tax_src_id" ref="account_tax_template_purchase_21"/>
+        <field name="tax_dest_id" ref="account_tax_template_purchase_0"/>
+    </record>
+</odoo>
```

Comment:
What profit taxes you are talking about? In comment link you pointed, we were talking about reverse taxes. And there it was simplified to use tax groups, to map reverse taxes, so it would be simpler for user to select it (because in reverse tax case, you would need to select positive and negative tax).

Reply:
My comment link points to your comment on the other review, where you had Tax Group `tax_group_pt` (Profit Tax) added with this localization module, but no taxes used this group. I'm trying to figure out, if you actually had these taxes added before. Quote:
> Looks like left overs.
 
I'm aware, that they are not mandatory. Accountants, when closing financial year, can create the journal entries for these taxes manually, but:
* Applying taxes automatically is much easier and safer (no need to use calculators - less errors)
* The classification would ease SAF-T generation (future-proofing)

Reply:
I checked our legacy COA, and those types are left overs. Though, if such groups would be added, with which taxes it should be related?

Reply:
As Profit Taxes are not really related to existing Value Added Taxes, they should be added as separate taxes. At least this is how we have it with our localization module.